frontend/src/model/beta/settings/AccessRequestSettings.tsx

```diff
@@ -1,15 +1,44 @@
 import { LoadingButton } from '@mui/lab'
 import { Checkbox, Divider, FormControlLabel, Stack, Typography } from '@mui/material'
+import { patchModel } from 'actions/model'
 import { useState } from 'react'
 
-export default function AccessRequestSettings() {
-  const [allowUngoverned, setAllowUngoverned] = useState(false)
+import { ModelInterface } from '../../../../types/v2/types'
+import { getErrorMessage } from '../../../../utils/fetcher'
+import useNotification from '../../../common/Snackbar'
+import MessageAlert from '../../../MessageAlert'
+
+type ModelAccessProps = {
+  model: ModelInterface
+}
+
+export default function AccessRequestSettings({ model }: ModelAccessProps) {
+  const [allowUngoverned, setAllowUngoverned] = useState(model.settings.ungovernedAccess)
   const [loading, setLoading] = useState(false)
+  const [errorMessage, setErrorMessage] = useState('')
+  const sendNotification = useNotification()
 
-  const handleSave = () => {
+  async function handleSave() {
     setLoading(true)
 
-    //TODO - Save settings API request and setLoading(false) on error
+    const updatedModelSettings = {
+      settings: {
+        ungovernedAccess: allowUngoverned,
+      },
+    }
+
+    const response = await patchModel(model.id, updatedModelSettings)
+
+    if (!response.ok) {
+      setErrorMessage(await getErrorMessage(response))
+    } else {
+      sendNotification({
+        variant: 'success',
+        msg: 'Access updated',
```

Comment:
Change to something like "Access request settings updated" or something along those lines